src/main/game/Board.java

```diff
@@ -38,6 +38,17 @@ public List updateMove(String move, String playerType) {
         return rows;
     }
 
+    public List<Line> getRowLines() {
```

Comment:
I think a better mor the generic name here would be `rows` or maybe `getRows`. Same for the columns and diagonals methods.